From 6355e9a59904072ed995e936738e882e15d7998f Mon Sep 17 00:00:00 2001
From: Neil Roberts <neil@linux.intel.com>
Date: Thu, 15 Mar 2012 16:00:14 +0000
Subject: [PATCH 3/4] Don't link directly to libGL

Nothing in the Gnome Shell plugin now directly uses symbols from libGL
so we don't need to link to it. This helps to make the plugin support
other Cogl drivers and winsys's than GL+GLX (such as GLES+EGL).
---
 configure.ac |    3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

Index: gnome-shell-3.4.1/configure.ac
===================================================================
--- gnome-shell-3.4.1.orig/configure.ac	2012-05-28 20:42:15.951114944 +0200
+++ gnome-shell-3.4.1/configure.ac	2012-05-28 20:42:18.887114820 +0200
@@ -50,7 +50,7 @@
    AC_MSG_RESULT(yes)
    build_recorder=true
    recorder_modules="gstreamer-0.10 gstreamer-base-0.10 x11"
-   PKG_CHECK_MODULES(TEST_SHELL_RECORDER, $recorder_modules clutter-1.0 xfixes gl)
+   PKG_CHECK_MODULES(TEST_SHELL_RECORDER, $recorder_modules clutter-1.0 xfixes)
 else
    AC_MSG_RESULT(no)
 fi
@@ -82,7 +82,6 @@
                                gjs-internals-1.0 >= $GJS_MIN_VERSION
 			       libgnome-menu-3.0 $recorder_modules
                                gdk-x11-3.0 libsoup-2.4
-                               gl
 			       clutter-x11-1.0 >= $CLUTTER_MIN_VERSION
 			       clutter-glx-1.0 >= $CLUTTER_MIN_VERSION
                                libstartup-notification-1.0 >= $STARTUP_NOTIFICATION_MIN_VERSION
